requirements.txt

```diff
@@ -0,0 +1,6 @@
+virtualenv
```

Comment:
I am getting confused now. We don't need to `pip install virtualenv` in the Heroku app. 
AFAIK, the requirements.txt in root directory is required only for Heroku